homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
+  # Bronze
+  config-flow: done
+  brands: done
+  dependency-transparency: done
+  common-modules: done
+  has-entity-name: done
+  action-setup:
+    status: exempt
+    comment: The integration does not register actions.
+  appropriate-polling: done
+  test-before-configure: done
+  entity-event-setup: done
+  unique-config-entry: done
+  entity-unique-id: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  test-before-setup: done
+  docs-high-level-description: done
+  config-flow-test-coverage: done
+  docs-actions:
+    status: exempt
+    comment: The integration does not register actions.
+  runtime-data: done
+
+  # Silver
+  log-when-unavailable: done
+  config-entry-unloading: done
+  reauthentication-flow: done
+  action-exceptions:
+    status: exempt
+    comment: The integration does not register actions.
+  docs-installation-parameters: done
+  integration-owner: done
+  parallel-updates: todo
+  test-coverage: todo
```

Comment:
in test_init.py, let's replace `    assert await async_setup_component(hass, DOMAIN, {})` with the `hass.config_entries.async_setup()` one

Reply:
https://github.com/home-assistant/core/pull/143398